app/views/posts/show.html.slim

```diff
@@ -1,17 +1,41 @@
+= stylesheet_link_tag 'comments'
+
 .dflex.justify-content-center
   .card style="padding:2% 5% 2% 5%"
-    small.text-muted
-      | Posted by&#160
-      i
+    span
+      small.text-muted
+        | Posted by
         = User.find(@post.user_id).username
```

Comment:
A kādēļ neizmanto asociāciju? `@post.user.username` ?

Reply:
Šeit starp citu tev ir N+1 queries problēma. tu vēl par to neko nezini, tas fine, kkas nākotnē pastāstīsim.

Idejiski sakties - ja tev būs 50 komentāri, tad tu taisīsi 50 User.find, no kuriem daži pat var būt tie paši useri.

kruta būtu visus userus ielasīt ar 1 queriju. Par laimi rails to piedāvā pie kokrētās relācijas papildus jāpieiveno `preload(:user)` un tas arī viss :D -> 
```ruby
@post = Post.find(id)
@comments  = @post.comments.preload(:user)
@comments.each {|c| c.user.username} # no N+1 queries :)
```
izņemsi to `preload(:user)` un redzēs rails logā N `SELECT * FROM users WHERE id = ? ["id", x]` SQL pieprasījumus

Reply:
oj,  tas N+1 prikols man te mazliet sajuka. ^ augšējais komentārs vairāk attiecas uz komentāriem :)

Bet šeit tāpāt paliek spēkā `@post.user.username`

Reply:
Piemirsu par to asociaciju iespeju uz otru pusi , man vina kka prata kustejas tikai viena virziena 😄